packages/default/scss/button/_layout.scss

```
@@ -80,6 +80,15 @@
             }
         }
 
+        .k-ie9 &,
```

Flexbox issues should be resolved like this

```suggestion
        .k-no-flexbox {
             .k-button {
                display: inline-block;
            }
        }
```

In addition, you could move the selector at the end of the file and add some handling for buton-group as well.